HLTrigger/special/src/HLTPixelActivityHFSumEnergyFilter.cc

```diff
@@ -0,0 +1,129 @@
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/stream/EDFilter.h"
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Utilities/interface/StreamID.h"
+
+#include "DataFormats/SiPixelCluster/interface/SiPixelCluster.h"
+#include "DataFormats/Common/interface/DetSetVector.h"
+#include "DataFormats/SiPixelDigi/interface/PixelDigi.h"
+#include "DataFormats/HcalRecHit/interface/HcalRecHitCollections.h"
+
+//
+// class declaration
+//
+
+class HLTPixelActivityHFSumEnergyFilter : public edm::stream::EDFilter<> {
```

Comment:
Here it is a EDFilter -- ???

Please separate properly the .h from the .cc content.